src/Native/System.Native/pal_errno.cpp

```diff
@@ -339,6 +347,7 @@ extern "C" int32_t SystemNative_ConvertErrorPalToPlatform(Error error)
         case PAL_EXDEV:
             return EXDEV;
         case PAL_ENONSTANDARD:
+        default:
```

Comment:
@stephentoub, thanks for the details. Actually i was suppressing the error not the warning, since these are considered as compiler error as we have -Weverything and -Werror switches in place.

In my research, these errno (131 et al.) are not handled by NetBSD, and therefore they are unavailable in enumeration. @krytarowski and other core NetBSD developers can provide us the specific details that how returning -1 versus 131 (both being equally unknown error numbers) would make a difference.

I totally agree with the idea of ifdef'ing out the corresponding PAL_ variants as well. I will fix it up shortly.